Tests/.swiftlint.yml

```diff
@@ -0,0 +1,9 @@
+disabled_rules:
+  - force_cast
+  - force_try
+  - file_length
+  - function_body_length
+  - identifier_name
+  - line_length
+  - type_body_length
+  - type_name
```

Comment:
Same q as in PR 3 re between repos, although defo care less for tests

Reply:
Let's chat about it in PR 3, since I agree that the test lint rules matter less - it's the top level lint file that matters most of all to me, that should be consistent everywhere.